clientapi/auth/storage/accounts/postgres/account_data_table.go

```diff
@@ -90,7 +91,7 @@ func (s *accountDataStatements) selectAccountData(
 	if err != nil {
 		return
 	}
-	defer rows.Close() // nolint: errcheck
+	defer common.LogIfError(ctx, rows.Close(), "rows.close() failed")
```

Comment:
Contextual error messages would help here e.g by embedding the name of the calling function:
```
	defer common.LogIfError(ctx, rows.Close(), "insertAccountData: rows.close() failed")
```

Reply:
everywhere or just this one?

Reply:
Everywhere please! (sorry I know it's a pain)